### PR TITLE
Fix/xxx  gltf export

### PR DESCRIPTION
**Pull request by inkhey**
Replacement for https://github.com/algoo/preview-generator/pull/195 (merged with current develop status)

This add a new 3d format support: gltf.

To test this: verify new and old 3d format support.

## Implemented solution
- add experimental gltf support enablable with env var.

### CODE REVIEW

**Comment by inkhey** [comment]
I found some issue with non-embedded .gltf file that cause segfault, here https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/MaterialsVariantsShoe/glTF :. A proper fix, or an encapsulation of vtk may be a solution.

File: tests/input/gltf/test_gltf_vtk.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import typing
+
+from PIL import Image
+import pytest
+
+from preview_generator.preview.builder.cad__vtk import ImagePreviewBuilderVtk
+from preview_generator.utils import ImgDims
+
+CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
+CACHE_DIR = "/tmp/preview-generator-tests/cache/"
```
Comment (raphj) [inline tests/input/gltf/test_gltf_vtk.py:15]: Shouldn't we use a temporary directory (using mktemp)?

**Review by raphj — APPROVED** [review]

File: tests/input/gltf/test_gltf_vtk.py (hunk 1 above)
Comment (inkhey) [reply tests/input/gltf/test_gltf_vtk.py:15]: probably. We need to refactor tests code.

Comment (raphj) [reply tests/input/gltf/test_gltf_vtk.py:15]: https://github.com/algoo/preview-generator/issues/243

**Pull request merged**